shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmap.java

```diff
@@ -545,6 +545,7 @@ protected int getHeight() {
   @Implementation
```

Comment:
Can you remove this method (and getDensity below). It is better to use the real implementation if possible. Also remove the `private int density;` member and you may need to fix a couple of times that `density` is set.